test/unit/tags/comment_tag_unit_test.rb

```diff
@@ -31,25 +31,45 @@ def test_does_not_parse_nodes_inside_a_comment
     LIQUID
   end
 
-  def test_allows_incomplete_tags_inside_a_comment
-    assert_template_result("", <<~LIQUID.chomp)
+  def test_allows_unclosed_tags
+    assert_template_result('', <<~LIQUID.chomp)
       {% comment %}
-        {% assign foo = "1"
+        {% if true %}
       {% endcomment %}
     LIQUID
+  end
 
-    assert_template_result("", <<~LIQUID.chomp)
+  def test_block_body_must_be_valid
```

Comment:
The new name for this test is a bit confusing. I'm not sure what you mean by "block body must be valid".

Reply:
right... this unit test is testing the comment tag's parsing behaviour with open tags like `{% assign a=1`
I have renamed the unit test to `test_open_tags_in_comment`